File: CHPT/source/trunk/supp_app/SspDispatchService/src/com/ctfo/storage/model/permission/SysUser.java
package com.ctfo.storage.model.permission;

import java.io.Serializable;

/**
 * 
 * 
 * <p>
 * ----------------------------------------------------------------------------- <br>
 * 工程名 ： SspDispatchService <br>
 * 功能： 人员信息<br>
 * 描述： 人员信息<br>
 * 授权 : (C) Copyright (c) 2011 <br>
 * 公司 : 北京中交慧联信息科技有限公司 <br>
 * ----------------------------------------------------------------------------- <br>
 * 修改历史 <br>
 * <table width="432" border="1">
 * <tr>
 * <td>版本</td>
 * <td>时间</td>
 * <td>作者</td>
 * <td>改变</td>
 * </tr>
 * <tr>
 * <td>1.0</td>
 * <td>2014-11-4</td>
 * <td>xuehui</td>
 * <td>创建</td>
 * </tr>
 * </table>
 * <br>
 * <font color="#FF0000">注意: 本内容仅限于[北京中交慧联信息科技有限公司]内部使用，禁止转发</font> <br>
 * 
 * @version 1.0
 * 
 * @author xuehui
 * @since JDK1.6
 */
public class SysUser implements Serializable {

	/** */
	private static final long serialVersionUID = 8153377437741726611L;

	/** 人员id */
	private String userId;

	/** 所属组织，关联组织机构表 */
	private String orgId;

	/** 人员编码 */
	private String userCode;

	/** 姓名 */
	private String userName;

	/** 登陆账号名称 */
	private String landName;

	/** 性别，关联字典码表 */
	private String sex;

	/** 民族，关联字典码表 */
	private String nation;

	/** 出生日期 */
	private Long birthday;

	/** 证件类型，关联字典码表 */
	private String idcardType;

	/** 证件号码 */
	private String idcardNum;

	/** 户口所在地 */
	private String registerAddress;

	/** 籍贯 */
	private String nativePlace;

	/** 政治面貌，关联字典码表 */
	private String politicalStatus;

	/** 手机 */
	private String userPhone;

	/** 固话 */
	private String userTelephone;

	/** 传真 */
	private String userFax;

	/** 邮箱 */
	private String userEmail;

	/** 联系地址 */
	private String userAddress;

	/** 身高 */
	private String userHeight;

	/** 体重 */
	private String userWeight;

	/** 入职日期 */
	private Long entryDate;

	/** 职务，关联字典码表 */
	private String post;

	/** 岗位，关联字典码表 */
	private String position;

	/** 级别，关联字典码表 */
	private String level;

	/** 毕业院校 */
	private String graduateInstitutions;

	/** 专业 */
	private String specialty;

	/** 学历，关联字典码表 */
	private String education;

	/** 毕业时间 */
	private Long graduateDate;

	/** 技术特长 */
	private String technicalExpertise;

	/** 工资 */
	private String wage;

	/** 是否在线，0否，1是 */
	private String isOnline;

	/** 是否操作员，0否，1是 */
	private String isOperator;

	/** 密码 */
	private String password;

	/** 删除标记，0删除，1未删除 */
	private String enableFlag;

	/** 备注 */
	private String remark;

	/** 状态，关联字典码表 */
	private String status;

	/** 数据来源，关联字典码表 */
	private String dataSources;

	/** 创建人 */
	private String createBy;

	/** 创建时间 */
	private Long createTime;

	/** 最后编辑人 */
	private String updateBy;

	/** 最后编辑时间 */
	private Long updateTime;

	/** 服务站id，云平台用 */
	private String serStationId;

	/** 帐套id，云平台用 */
	private String setBookId;

	/** 联系人CRMid */
	private String contCrmGuid;

	public String getUserId() {
		return userId;
	}

	public void setUserId(String userId) {
		this.userId = userId;
	}

	public String getOrgId() {
		return orgId;
	}

	public void setOrgId(String orgId) {
		this.orgId = orgId;
	}

	public String getUserCode() {
		return userCode;
	}

	public void setUserCode(String userCode) {
		this.userCode = userCode;
	}

	public String getUserName() {
		return userName;
	}

	public void setUserName(String userName) {
		this.userName = userName;
	}

	public String getLandName() {
		return landName;
	}

	public void setLandName(String landName) {
		this.landName = landName;
	}

	public String getSex() {
		return sex;
	}

	public void setSex(String sex) {
		this.sex = sex;
	}

	public String getNation() {
		return nation;
	}

	public void setNation(String nation) {
		this.nation = nation;
	}

	public Long getBirthday() {
		return birthday;
	}

	public void setBirthday(Long birthday) {
		this.birthday = birthday;
	}

	public String getIdcardType() {
		return idcardType;
	}

	public void setIdcardType(String idcardType) {
		this.idcardType = idcardType;
	}

	public String getIdcardNum() {
		return idcardNum;
	}

	public void setIdcardNum(String idcardNum) {
		this.idcardNum = idcardNum;
	}

	public String getRegisterAddress() {
		return registerAddress;
	}

	public void setRegisterAddress(String registerAddress) {
		this.registerAddress = registerAddress;
	}

	public String getNativePlace() {
		return nativePlace;
	}

	public void setNativePlace(String nativePlace) {
		this.nativePlace = nativePlace;
	}

	public String getPoliticalStatus() {
		return politicalStatus;
	}

	public void setPoliticalStatus(String politicalStatus) {
		this.politicalStatus = politicalStatus;
	}

	public String getUserPhone() {
		return userPhone;
	}

	public void setUserPhone(String userPhone) {
		this.userPhone = userPhone;
	}

	public String getUserTelephone() {
		return userTelephone;
	}

	public void setUserTelephone(String userTelephone) {
		this.userTelephone = userTelephone;
	}

	public String getUserFax() {
		return userFax;
	}

	public void setUserFax(String userFax) {
		this.userFax = userFax;
	}

	public String getUserEmail() {
		return userEmail;
	}

	public void setUserEmail(String userEmail) {
		this.userEmail = userEmail;
	}

	public String getUserAddress() {
		return userAddress;
	}

	public void setUserAddress(String userAddress) {
		this.userAddress = userAddress;
	}

	public String getUserHeight() {
		return userHeight;
	}

	public void setUserHeight(String userHeight) {
		this.userHeight = userHeight;
	}

	public String getUserWeight() {
		return userWeight;
	}

	public void setUserWeight(String userWeight) {
		this.userWeight = userWeight;
	}

	public Long getEntryDate() {
		return entryDate;
	}

	public void setEntryDate(Long entryDate) {
		this.entryDate = entryDate;
	}

	public String getPost() {
		return post;
	}

	public void setPost(String post) {
		this.post = post;
	}

	public String getPosition() {
		return position;
	}

	public void setPosition(String position) {
		this.position = position;
	}

	public String getLevel() {
		return level;
	}

	public void setLevel(String level) {
		this.level = level;
	}

	public String getGraduateInstitutions() {
		return graduateInstitutions;
	}

	public void setGraduateInstitutions(String graduateInstitutions) {
		this.graduateInstitutions = graduateInstitutions;
	}

	public String getSpecialty() {
		return specialty;
	}

	public void setSpecialty(String specialty) {
		this.specialty = specialty;
	}

	public String getEducation() {
		return education;
	}

	public void setEducation(String education) {
		this.education = education;
	}

	public Long getGraduateDate() {
		return graduateDate;
	}

	public void setGraduateDate(Long graduateDate) {
		this.graduateDate = graduateDate;
	}

	public String getTechnicalExpertise() {
		return technicalExpertise;
	}

	public void setTechnicalExpertise(String technicalExpertise) {
		this.technicalExpertise = technicalExpertise;
	}

	public String getWage() {
		return wage;
	}

	public void setWage(String wage) {
		this.wage = wage;
	}

	public String getIsOnline() {
		return isOnline;
	}

	public void setIsOnline(String isOnline) {
		this.isOnline = isOnline;
	}

	public String getIsOperator() {
		return isOperator;
	}

	public void setIsOperator(String isOperator) {
		this.isOperator = isOperator;
	}

	public String getPassword() {
		return password;
	}

	public void setPassword(String password) {
		this.password = password;
	}

	public String getEnableFlag() {
		return enableFlag;
	}

	public void setEnableFlag(String enableFlag) {
		this.enableFlag = enableFlag;
	}

	public String getRemark() {
		return remark;
	}

	public void setRemark(String remark) {
		this.remark = remark;
	}

	public String getStatus() {
		return status;
	}

	public void setStatus(String status) {
		this.status = status;
	}

	public String getDataSources() {
		return dataSources;
	}

	public void setDataSources(String dataSources) {
		this.dataSources = dataSources;
	}

	public String getCreateBy() {
		return createBy;
	}

	public void setCreateBy(String createBy) {
		this.createBy = createBy;
	}

	public Long getCreateTime() {
		return createTime;
	}

	public void setCreateTime(Long createTime) {
		this.createTime = createTime;
	}

	public String getUpdateBy() {
		return updateBy;
	}

	public void setUpdateBy(String updateBy) {
		this.updateBy = updateBy;
	}

	public Long getUpdateTime() {
		return updateTime;
	}

	public void setUpdateTime(Long updateTime) {
		this.updateTime = updateTime;
	}

	public String getSerStationId() {
		return serStationId;
	}

	public void setSerStationId(String serStationId) {
		this.serStationId = serStationId;
	}

	public String getSetBookId() {
		return setBookId;
	}

	public void setSetBookId(String setBookId) {
		this.setBookId = setBookId;
	}

	public String getContCrmGuid() {
		return contCrmGuid;
	}

	public void setContCrmGuid(String contCrmGuid) {
		this.contCrmGuid = contCrmGuid;
	}

}

File: CHPT/source/tags/V0.3.4/Model/sys_user.cs
using System;
namespace Model
{
	/// <summary>
	/// 人员信息
	/// </summary>
	[Serializable]
	public partial class sys_user
	{
		public sys_user()
		{}
		#region Model
		private string _user_id;
		private string _org_id;
		private string _user_code;
		private string _user_name;
		private string _land_name;
		private string _sex;
		private string _nation;
		private long? _birthday;
		private string _idcard_type;
		private string _idcard_num;
		private string _register_address;
		private string _native_place;
		private string _political_status;
		private string _user_phone;
		private string _user_telephone;
		private string _user_fax;
		private string _user_email;
		private string _user_address;
		private string _user_height;
		private string _user_weight;
		private long? _entry_date;
		private string _post;
		private string _position;
		private string _level;
		private string _graduate_institutions;
		private string _specialty;
		private string _education;
		private long? _graduate_date;
		private string _technical_expertise;
		private string _wage;
		private string _is_operator;
		private string _password;
		private string _remark;
		private string _status;
		private string _data_sources;
		private string _create_by;
		private long _create_time;
		private string _update_by;
		private long? _update_time;
		/// <summary>
		/// 人员id
		/// </summary>
		public string user_id
		{
			set{ _user_id=value;}
			get{return _user_id;}
		}
		/// <summary>
		/// 所属组织，关联组织机构表
		/// </summary>
		public string org_id
		{
			set{ _org_id=value;}
			get{return _org_id;}
		}
		/// <summary>
		/// 人员编码
		/// </summary>
		public string user_code
		{
			set{ _user_code=value;}
			get{return _user_code;}
		}
		/// <summary>
		/// 姓名
		/// </summary>
		public string user_name
		{
			set{ _user_name=value;}
			get{return _user_name;}
		}
		/// <summary>
		/// 登陆账号名称
		/// </summary>
		public string land_name
		{
			set{ _land_name=value;}
			get{return _land_name;}
		}
		/// <summary>
		/// 性别，关联字典码表
		/// </summary>
		public string sex
		{
			set{ _sex=value;}
			get{return _sex;}
		}
		/// <summary>
		/// 民族，关联字典码表
		/// </summary>
		public string nation
		{
			set{ _nation=value;}
			get{return _nation;}
		}
		/// <summary>
		/// 出生日期
		/// </summary>
		public long? birthday
		{
			set{ _birthday=value;}
			get{return _birthday;}
		}
		/// <summary>
		/// 证件类型，关联字典码表
		/// </summary>
		public string idcard_type
		{
			set{ _idcard_type=value;}
			get{return _idcard_type;}
		}
		/// <summary>
		/// 证件号码
		/// </summary>
		public string idcard_num
		{
			set{ _idcard_num=value;}
			get{return _idcard_num;}
		}
		/// <summary>
		/// 户口所在地
		/// </summary>
		public string register_address
		{
			set{ _register_address=value;}
			get{return _register_address;}
		}
		/// <summary>
		/// 籍贯
		/// </summary>
		public string native_place
		{
			set{ _native_place=value;}
			get{return _native_place;}
		}
		/// <summary>
		/// 政治面貌，关联字典码表
		/// </summary>
		public string political_status
		{
			set{ _political_status=value;}
			get{return _political_status;}
		}
		/// <summary>
		/// 手机
		/// </summary>
		public string user_phone
		{
			set{ _user_phone=value;}
			get{return _user_phone;}
		}
		/// <summary>
		/// 固话
		/// </summary>
		public string user_telephone
		{
			set{ _user_telephone=value;}
			get{return _user_telephone;}
		}
		/// <summary>
		/// 传真
		/// </summary>
		public string user_fax
		{
			set{ _user_fax=value;}
			get{return _user_fax;}
		}
		/// <summary>
		/// 邮箱
		/// </summary>
		public string user_email
		{
			set{ _user_email=value;}
			get{return _user_email;}
		}
		/// <summary>
		/// 联系地址
		/// </summary>
		public string user_address
		{
			set{ _user_address=value;}
			get{return _user_address;}
		}
		/// <summary>
		/// 身高
		/// </summary>
		public string user_height
		{
			set{ _user_height=value;}
			get{return _user_height;}
		}
		/// <summary>
		/// 体重
		/// </summary>
		public string user_weight
		{
			set{ _user_weight=value;}
			get{return _user_weight;}
		}
		/// <summary>
		/// 入职日期
		/// </summary>
		public long? entry_date
		{
			set{ _entry_date=value;}
			get{return _entry_date;}
		}
		/// <summary>
		/// 职务，关联字典码表
		/// </summary>
		public string post
		{
			set{ _post=value;}
			get{return _post;}
		}
		/// <summary>
		/// 岗位，关联字典码表
		/// </summary>
		public string position
		{
			set{ _position=value;}
			get{return _position;}
		}
		/// <summary>
		/// 级别，关联字典码表
		/// </summary>
		public string level
		{
			set{ _level=value;}
			get{return _level;}
		}
		/// <summary>
		/// 毕业院校
		/// </summary>
		public string graduate_institutions
		{
			set{ _graduate_institutions=value;}
			get{return _graduate_institutions;}
		}
		/// <summary>
		/// 专业
		/// </summary>
		public string specialty
		{
			set{ _specialty=value;}
			get{return _specialty;}
		}
		/// <summary>
		/// 学历，关联字典码表
		/// </summary>
		public string education
		{
			set{ _education=value;}
			get{return _education;}
		}
		/// <summary>
		/// 毕业时间
		/// </summary>
		public long? graduate_date
		{
			set{ _graduate_date=value;}
			get{return _graduate_date;}
		}
		/// <summary>
		/// 技术特长
		/// </summary>
		public string technical_expertise
		{
			set{ _technical_expertise=value;}
			get{return _technical_expertise;}
		}
		/// <summary>
		/// 工资
		/// </summary>
		public string wage
		{
			set{ _wage=value;}
			get{return _wage;}
		}
		/// <summary>
		/// 是否操作员，0否，1是
		/// </summary>
		public string is_operator
		{
			set{ _is_operator=value;}
			get{return _is_operator;}
		}
		/// <summary>
		/// 密码
		/// </summary>
		public string password
		{
			set{ _password=value;}
			get{return _password;}
		}
		/// <summary>
		/// 备注
		/// </summary>
		public string remark
		{
			set{ _remark=value;}
			get{return _remark;}
		}
		/// <summary>
		/// 状态，关联字典码表
		/// </summary>
		public string status
		{
			set{ _status=value;}
			get{return _status;}
		}
		/// <summary>
		/// 数据来源，关联字典码表
		/// </summary>
		public string data_sources
		{
			set{ _data_sources=value;}
			get{return _data_sources;}
		}
		/// <summary>
		/// 创建人
		/// </summary>
		public string create_by
		{
			set{ _create_by=value;}
			get{return _create_by;}
		}
		/// <summary>
		/// 创建时间
		/// </summary>
		public long create_time
		{
			set{ _create_time=value;}
			get{return _create_time;}
		}
		/// <summary>
		/// 最后编辑人
		/// </summary>
		public string update_by
		{
			set{ _update_by=value;}
			get{return _update_by;}
		}
		/// <summary>
		/// 最后编辑时间
		/// </summary>
		public long? update_time
		{
			set{ _update_time=value;}
			get{return _update_time;}
		}
		#endregion Model

	}
}

